### PR TITLE
[main] TAB-9521:remove unused main methods from codebase

### PR DESCRIPTION
**Pull request by devlg75**

### CODE REVIEW

File: terracotta-kit/src/test/java/org/terracotta/kit/EhcacheAllIT.java [hunk 1]
```diff
@@ -33,7 +34,7 @@ public void testEhcacheAllManifest() throws IOException, ClassNotFoundException
         System.out.println(path.toAbsolutePath());
 
         try (URLClassLoader urlClassLoader = URLClassLoader.newInstance(new URL[] {path.toUri().toURL()}, null)) {
-            urlClassLoader.loadClass("org.terracotta.util.ToolkitVersion");
+            Assert.assertNotNull(urlClassLoader.findResource("/Version.info"));
```
Comment (cljohnso) [inline terracotta-kit/src/test/java/org/terracotta/kit/EhcacheAllIT.java:37]: Just a reminder to alter this ...

**Review by cljohnso — CHANGES_REQUESTED** [review]
Looks good other than the `/` you noted in chat.

This commit stream needs to go into main, 4.4.1, 4.4.0, and possibly 4.3.10.  Is this commit stream properly daggy?

File: terracotta-kit/src/test/java/org/terracotta/kit/EhcacheAllIT.java (hunk 1 above)
Comment (devlg75) [reply terracotta-kit/src/test/java/org/terracotta/kit/EhcacheAllIT.java]: done.

**Comment by devlg75** [comment]
> Looks good other than the `/` you noted in chat.
> 
> This commit stream needs to go into main, 4.4.1, 4.4.0, and possibly 4.3.10. Is this commit stream properly daggy?

this commit is daggy, I'll raise PR's for other branches as well.

**Review by cljohnso — APPROVED** [review]

**Pull request merged**